cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -131,7 +131,7 @@ describe('CourseOutlinePage', function() {
             group_access: {},
             user_partition_info: {},
             hide_from_toc: false,
-            enable_hide_from_toc_ui: true
+            enable_hide_from_toc_ui: false,
```

Comment:
I re-reviewed this and found this change. Do you think it's necessary? If not, can we remove it?

Reply:
This change is not necessary, I will leave it as it was.